process/peer/process.go

```diff
@@ -977,11 +977,6 @@ func (vs *validatorStatistics) savePeerAccountData(
 		return err
 	}
 
-	err = peerAccount.SetBLSPublicKey(node.PubKeyBytes())
-	if err != nil {
```

Comment:
so we loadPeerAccount from above it will come will BLSPublicKey set correctly?

Reply:
Yes.